### ISSUE TITLE
Memoized tasks get no task_time_running

### ISSUE THREAD

**Issue by benclifford** [Parsl/parsl#1596]
**Describe the bug**
Tasks where the result come from memoization have no task_time_running field set, although task_time_submitted and task_time_result are set.

This is weird although arguably acceptable:

```
sqlite> select task_time_submitted, task_time_running, task_time_returned from task where task_hashsum = '063832107cd7bc65da29b00acf107ab7';
task_time_submitted|task_time_running|task_time_returned
2020-03-05 06:21:38.786590|2020-03-05 06:21:38.860503|2020-03-05 06:22:49.782674
2020-03-05 06:33:40.916047||2020-03-05 06:33:40.916967
2020-03-05 07:04:57.007578||2020-03-05 07:04:57.008405
2020-03-05 07:46:54.452044||2020-03-05 07:46:54.454449
2020-03-05 08:36:01.212502||2020-03-05 08:36:01.213419
2020-03-05 10:07:23.631465||2020-03-05 10:07:23.633054
2020-03-05 10:53:44.910399||2020-03-05 10:53:44.911300
2020-03-06 02:41:13.160961||2020-03-06 02:41:13.163175
```

**To Reproduce**
Run multiple runs that restore tasks from a checkpoint. Pick one such task and make an sql query as above. Observe the first run, which runs through an executor, has all three fields; and the others which come from memoization/checkpointing do not.

**Expected behavior**
Task rows should have a task_time_running time. This might end up being around the same as task_time_returned, but not necessarily - that's making the assumption that memo lookup takes a trivial amount of time.

**Environment**
lsst-dm-202002 branch of parsl

**Comment by ZhuozhaoLi** [comment]
In monitoring, the `task_time_running` is inferred by the first `RESOURCE` message from the `monitoring_wrapper`.
For memoized tasks, we probably just make it equal to `task_time_submitted`

**Comment by benclifford** [comment]
Using task_time_running = task_time_submitted will give memoized tasks with dependencies a potentially long run time rather than 0 run time, and make them potentially appear to start running before their dependencies.

**Comment by benclifford** [comment]
This issue should be revisited after/as part of #1668 LSST merge work

**Comment by benclifford** [comment]
PR #1808 makes some changes to how this data is represented. In that PR, a memoized task does not have a `try` table entry, and so there are no try timing fields to be left empty. If merged, PR #1808 probably closes this issue.

**Comment by benclifford** [comment]
The schema was rearranged in #1808 and this is now not an issue - the empty field has disappeared from the task table, and become part of the try table, which does not have any rows for memoized tasks.
